pixel/head.html

```diff
@@ -1,12 +1,25 @@
-<script async type="text/javascript" src="https://t.profitshare.ro/files_shared/tr/{{settings.cookieScriptName}}.js"></script>
+<script>
+  (function() {
+    var domain = "{{ settings.cookieScriptDomain }}" || "t.profitshare.ro";
+    var src = `https://${domain}/files_shared/tr/{{settings.cookieScriptName}}.js`
+    var s;
+        s = document.createElement('script');
+        s.type="text/javascript";
+        s.src = src;
+        document.head.appendChild(s);
+  })();
+</script>
+
 <script>
 (function() {
     var clickCode = "{{ settings.clickCode }}";
     var advertiserCode = "{{ settings.advertiserCode }}";
     var key = "{{ settings.key }}";
     var cookieScriptName = "{{settings.cookieScriptName}}";
+    var cookieScriptDomain = "{{settings.cookieScriptDomain}}";
+    var iframeDomain = "{{settings.iframeDomain}}";
     
-    if (!clickCode || !advertiserCode || !key || !cookieScriptName ) {
+    if (!clickCode || !advertiserCode || !key || !cookieScriptName || !cookieScriptDomain || !iframeDomain ) {
```

Comment:
I would change this and if cookieScriptDomain or iframeDomain are not set, to log only an info or warning, something like: "Missing cookieScriptDomain or iframeDomain. Default values will be applied." 

I'm saying this, because those two new configs have a fallback to the default value (for backwards compatibility reasons) if they are not.